<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Gav Wood</author>
 <class>KCMSync</class>
 <widget class="QWidget" name="KCMSync">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Akunambol sync</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <widget class="QPushButton" name="pbAddServer">
     <property name="text">
      <string>Add...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pbRemoveServer">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pbEditServer">
     <property name="text">
      <string>Edit...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="6">
    <widget class="QListWidget" name="lwServerList">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pbSyncNow">
     <property name="text">
      <string>Sync now</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QProgressBar" name="pbSyncstate">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>252</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <includes>
  <include location="local">kpushbutton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
